examples/ads.amp.html

```diff
@@ -539,6 +540,17 @@ <h2 id="industrybrains">Industrybrains</h2>
     <div fallback></div>
   </amp-ad>
 
+  <h2 id="inmobi">InMobi</h2>
+  <amp-ad
+    width="320"
```

Comment:
nitpick, to keep a consistency, could you move `width` and `height` to the previous line?